pom.xml

```diff
@@ -273,11 +273,11 @@
             <scope>provided</scope>
         </dependency>
         <dependency>
-            <groupId>org.apache.commons</groupId>
-            <artifactId>commons-lang3</artifactId>
-            <version>3.6</version>
-            <scope>provided</scope>
-        </dependency>
+			<groupId>org.apache.commons</groupId>
```

Comment:
nit: please make sure they indentation does not change compared to the other entries in the pom.xml file.